src/main/java/com/pankaj/tictactoegame/service/ITicTacToeService.java

```diff
@@ -0,0 +1,11 @@
+package com.pankaj.tictactoegame.service;
+
+import com.pankaj.tictactoegame.models.Player;
+
+public interface ITicTacToeService {
```

Comment:
[For my knowledge] what is the motivation of creating an interface for this, I can't think of any variation which may implement this in future?